prusti-interface/src/utils.rs

```diff
@@ -307,6 +307,10 @@ pub fn has_extern_spec_attr(attrs: &[ast::Attribute]) -> bool {
     has_prusti_attr(attrs, "extern_spec")
 }
 
+pub fn has_model_attr(attrs: &[ast::Attribute]) -> bool {
+    has_prusti_attr(attrs, "model_generator")
```

Comment:
I don't like this name, we might be using Rust generators to model some things soonish :)

Reply:
This is a simple marked used to check that `.model()` is called from spec code only. Suggestion for new name: `to_model_fn`?

Reply:
`to_model_fn` or even just `to_model` should be good.